qiskit_addon_sqd/fermion.py

```diff
@@ -89,7 +88,7 @@ def solve_fermion(
             configurations should be specified by column indices in range ``(N/2, 0]``, where ``N``
             is the number of qubits.
 
-            (DEPRECATED) The configurations may also be specified by a length-2 tuple of sorted 1D
+            The configurations may also be specified by a length-2 tuple of sorted 1D
```

Comment:
Thanks Kevin, can we paste the dice solver description of this arg here for consistency?

Or if you like this one better, we can update it over there.

Reply:
Done.